regulations/templates/regulations/preamble-chrome.html

```diff
@@ -56,7 +56,7 @@ <h3 class="subpart-heading">{{cfr_part_toc.title}} CFR {{cfr_part_toc.part}}</h3
 {% block chrome-content %}
 <main role="main">
 
-  <div id="content-body" class="main-content">
+  <div id="content-body" class="main-content preamble-wrapper">
```

Comment:
That class wasn't used in the styles before, but it looks like @xtine's using it now. Which sounds fine to me.